.github/workflows/ci-test-controllers.yml

```diff
@@ -1,31 +1,61 @@
 name: ci-test-controllers
 
 on:
-  push:
```

Comment:
Why not test on main?

Reply:
It runs the ginkgo test. Just seemed redundant given that we already test it on PR and we have ginkgo tests running in other test suites. Also, the build tests are still run on main.